shared/templates/pam_account_password_faillock/tests/ubuntu_missing_pamd.fail.sh

```diff
@@ -1,9 +1,8 @@
 #!/bin/bash
 # platform = multi_platform_ubuntu
 
-source ubuntu_common.sh
+rm -f /usr/share/pam-configs/cac_faillock*
 
-sed -i '/pam_faillock\.so/d' /etc/pam.d/common-auth
-sed -i '/pam_faillock\.so/d' /etc/pam.d/common-account
+DEBIAN_FRONTEND=noninteractive pam-auth-update
 
 echo "deny=1" > /etc/security/faillock.conf
```

Comment:
The parameter `deny` should not be hardcoded in the test since it's not used in other rules that use the template:
- accounts_passwords_pam_faillock_unlock_time
- accounts_passwords_pam_faillock_interval

Use `{{{ PRM_NAME }}}`, `{{{ VARIABLE_UPPER_BOUND }}}`, and `{{{ VARIABLE_LOWER_BOUND }}}` instead.